nautobot_chatops_ipfabric/ipfabric_wrapper.py

```diff
@@ -0,0 +1,136 @@
+"""IPFabric API integration."""
+
+import logging
+
+from ipfabric_diagrams import IPFDiagram
+from ipfabric import IPFClient
+
+# Default IP Fabric API pagination limit
+DEFAULT_PAGE_LIMIT = 100
+LAST = "$last"
+PREV = "$prev"
+LAST_LOCKED = "$lastLocked"
+
+# URLs
+INVENTORY_DEVICES_URL = "tables/inventory/devices"
```

Comment:
my idea was to have then within the class definition. does it make sense to you?
(so you don't need to import them)

Reply:
Oh I like that. The importing was the reason I put them in the `worker.py`. Will update.